src/server/game/Spells/SpellEffects.cpp

```diff
@@ -2759,15 +2759,20 @@ void Spell::EffectTeleUnitsFaceCaster(SpellEffIndex  /*effIndex*/)
     if (!unitTarget || unitTarget->IsInFlight())
         return;
 
-    if (!m_targets.HasDst())
+    if (unitTarget->IsInFlight())
```

Comment:
It can also be deleted here. The above code already has this check

Reply:
Or I missed that totally, very true